.github/workflows/prepare-release.yml

```diff
@@ -12,8 +18,9 @@ jobs:
       # action is at https://github.com/realm/ci-actions/tree/main/update-changelog
       - name: Update Changelog
         id: update-changelog
-        uses: realm/ci-actions/update-changelog@v2
+        uses: realm/ci-actions/update-changelog@master
```

Comment:
Is this a deliberate change, i.e. we would rather use the master version of actions rather than released ones?

Reply:
I asked @nirinchev - it seems that's what other repositories are doing (and I didn't feel like pushing a new tag to the ci-actions repo 🙃 )